packages/odyssey-react-mui/src/Checkbox.tsx

```diff
@@ -38,30 +39,45 @@ const Checkbox = ({
   isDefaultChecked,
   isDisabled,
   isIndeterminate,
-  isOptional = true,
+  isRequired = true,
   label,
   name,
   onChange,
+  requiredLabel = "Required",
   value,
-}: CheckboxProps) => (
-  <FormControlLabel
-    aria-label={ariaLabel}
-    aria-labelledby={ariaLabelledBy}
-    aria-required={!isOptional}
-    checked={isChecked}
-    className={hasError ? "Mui-error" : ""}
-    control={
-      <MuiCheckbox indeterminate={isIndeterminate} required={!isOptional} />
-    }
-    defaultChecked={isDefaultChecked}
-    disabled={isDisabled}
-    label={label}
-    name={name}
-    onChange={onChange}
-    value={value}
-    required={!isOptional}
-  />
-);
+}: CheckboxProps) => {
+  let combinedLabel = <>{label}</>;
```

Comment:
You shouldn't need this fragment:

```suggestion
  let combinedLabel = label;
```

I'm assuming it's because of TS type.

And I absolutely don't like mutations. Why not use `useState` to manage this instead?